test_unit_triangle uint32 not safe to cast to int32 on 32-bit systems.

This is related to https://github.com/nschloe/meshplex/issues/90 and https://github.com/nschloe/meshplex/pull/94 .

uint32 was added to the dtypes tested with test_unit_triangle in test/mesh_tri/test_mesh_tri.py

It fails on 32.-bit systems, since

    Cannot cast array data from dtype('uint32') to dtype('int32') according to the rule 'safe'

see
https://ci.debian.net/data/autopkgtest/testing/i386/p/python-meshplex/8731225/log.gz
https://ci.debian.net/data/autopkgtest/testing/armhf/p/python-meshplex/8731019/log.gz

```
__________________________ test_unit_triangle[uint32] __________________________

cells_dtype = <class 'numpy.uint32'>

    @pytest.mark.parametrize("cells_dtype", cell_dtypes)
    def test_unit_triangle(cells_dtype):
        points = numpy.array([[0.0, 0.0], [1.0, 0.0], [0.0, 1.0]])
        cells = numpy.array([[0, 1, 2]], dtype=cells_dtype)
        mesh = meshplex.MeshTri(points, cells)
    
        tol = 1.0e-14
    
        assert (mesh.local_idx.T == [[1, 2], [2, 0], [0, 1]]).all()
        assert mesh.local_idx_inv == [[(0, 2), (1, 1)], [(0, 0), (1, 2)], [(0, 1), (1, 0)]]
    
        # ce_ratios
        assert near_equal(mesh.ce_ratios.T, [0.0, 0.5, 0.5], tol)
    
        # control volumes
>       assert near_equal(mesh.control_volumes, [0.25, 0.125, 0.125], tol)

test/mesh_tri/test_mesh_tri.py:58: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
meshplex/mesh_tri.py:346: in control_volumes
    return self.get_control_volumes()
meshplex/mesh_tri.py:335: in get_control_volumes
    self._control_volumes = numpy.bincount(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

args = (array([0, 1, 2], dtype=uint32),)
kwargs = {'minlength': 3, 'weights': array([0.25 , 0.125, 0.125])}
relevant_args = (array([0, 1, 2], dtype=uint32), array([0.25 , 0.125, 0.125]))

>   ???
E   TypeError: Cannot cast array data from dtype('uint32') to dtype('int32') according to the rule 'safe'

<__array_function__ internals>:5: TypeError
```

--- a/setup.cfg
+++ b/setup.cfg
@@ -1,6 +1,6 @@
 [metadata]
 name = meshplex
-version = 0.15.5
+version = 0.15.6
 author = Nico Schlömer
 author_email = nico.schloemer@gmail.com
 description = Fast tools for simplex meshes

--- a/test/mesh_tri/test_mesh_tri.py
+++ b/test/mesh_tri/test_mesh_tri.py
@@ -31,12 +31,12 @@ def _compute_polygon_area(pts):
 cell_dtypes = []
 cell_dtypes += [
     numpy.int32,
-    numpy.uint32,
 ]
 if platform.architecture()[0] == "64bit":
     cell_dtypes += [
+        numpy.uint32,  # when numpy is fixed, this can go to all arches
         numpy.int64,
-        # numpy.uint64
+        # numpy.uint64  # depends on the numpy fix
     ]
 
 

